pkg/tfsandbox/logger.go

```diff
@@ -0,0 +1,28 @@
+package tfsandbox
+
+import (
+	"context"
+
+	"github.com/pulumi/pulumi-terraform-bridge/v3/pkg/tfbridge"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type ctxKey struct{}
+
+// The key used to retrieve tfbridge.Logger from a context.
+var loggerCtxKey = ctxKey{}
+
+func WithLogger(ctx context.Context, logger *tfbridge.LogRedirector) context.Context {
```

Comment:
The exported fields here are a little strange to export from the `tfsandbox` package, perhaps a quick new package for these?